hoomd/md/EvaluatorPairMoliere.h

```diff
@@ -129,7 +159,8 @@ class EvaluatorPairMoliere
     protected:
         Scalar rsq;     //!< Stored rsq from the constructor
         Scalar rcutsq;  //!< Stored rcutsq from the constructor
-        Scalar Zsq;     //!< Zsq parameter extracted from the params passed to the constructor
+        Scalar qi;      //!< qi parameter extracted from the params passed to the constructor
+        Scalar qj;      //!< qj parameter extracted from the params passed to the constructor
```

Comment:
I don't think this should change. I misunderstood before. The params are stored elsewhere we only need `Zsq`.